server/app/services/languageProcessor/TaskMap.java

```diff
@@ -54,7 +56,11 @@ public JsonNode getTicketDescription(String issueKey, JsonNode responseBody) {
     if (Extractor.extractString(responseBody, "description").equals("[\"Issue Does Not Exist\"]")) {
       return parseToJson("fail", configuration.getString("error-message.issue-not-found"));
     } else {
-      String answer = "Description of " + issueKey + " is as follows: \n" +
+      String IssueId = responseBody.get("key").toString().replaceAll("\"", "");
```

Comment:
Please use camel case with a lower case first letter for variables. These are not constants, but constants should be all upper case with '_' between words, e.g. `SOME_CONSTANT_NAME`.